docs/new-infinite-scale-release-info.md

```diff
@@ -0,0 +1,33 @@
+# New Infinite Scale Releas Info
+
+If there is a new Infinite Scale release on the way, there are some things to know respectively to check:
+
+**Relevant for docs:**
+
+* The docs relevant releasing process in the `ocis` repo works the following:
+  * Based on a release tag like `v4.0.0`, a new `stable-x` branch is created.
```

Comment:
I think that we create a branch like `stable-4.0` - there is no `v` in the branch name. There is no patch version in the branch name. If what I said is correct, then detail can be added here:
```suggestion
  * Based on a release tag like `v4.0.0`, a new `stable-x` branch is created. For example, `stable-4.0`.
```